src/rpg/sweep/SweepParameter.tpp

```diff
@@ -47,11 +50,13 @@ namespace Rpg {
    template <int D>
    SweepParameter<D>::SweepParameter(System<D>& system)
     : type_(SweepParameter<D>::Null),
-      nID_(0),
+      nId_(0),
       id_(),
       initial_(0.0),
       change_(0.0),
-      systemPtr_(&system)
+      systemPtr_(0),
```

Comment:
Shouldn't this constructor store the address of the system?

Reply:
Oh yeah it should, good catch! This was just a mistake on my part with copying and pasting. I have no clue how all the unit tests managed to pass without this constructor being correct.

Reply:
I think the systemPtr_ is only used in the get_ and set_ functions, not in << and >> operators, and the unit tests may only test reading and writing.